### PR TITLE
feat/python: Fix test and lint to ci

### PR DESCRIPTION
**Pull request by kwek20**
Pyhton tests werent running on pytlint. fixed lint errors and added it

### CODE REVIEW

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
CI fails

**Comment by kwek20** [comment]
well look at that, caught myself with my own ci test addition :D

File: .github/workflows/bindings-python.yml [hunk 1]
```diff
@@ -121,6 +121,11 @@ jobs:
         working-directory: bindings/python
         run: tox -e lint-examples
 
+      - name: Run linter for tests
```
Comment (thibault-martinez) [inline .github/workflows/bindings-python.yml:120]: Hmm we should maybe do that to develop first and rebase

Comment (kwek20) [reply .github/workflows/bindings-python.yml:120]: Done

**Review by DaughterOfMars — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**